README.md

```diff
@@ -110,31 +113,42 @@ To find or create an access token for your organization, go to
 Follow these steps to configure your access token and ingest endpoint:
 
 1. Set `SIGNALFX_ACCESS_TOKEN` with your access token: 
-    ```bash
-     SIGNALFX_ACCESS_TOKEN=<access-token>
-    ```
+   ```bash
+   SIGNALFX_ACCESS_TOKEN=<access-token>
+   ```
 
 2. Set `SIGNALFX_ENDPOINT_URL` with your organization's realm:
-    ```bash
-     SIGNALFX_ENDPOINT_URL=https://ingest.<realm>.signalfx.com
-    ```
+   ```bash
+   SIGNALFX_ENDPOINT_URL=https://ingest.<realm>.signalfx.com
+   ```
 
    If you're sending data to an OpenTelemetry Collector, you have to specify
    the full path like this: `http://<otel-collector-host>:9411/api/v2/spans`
 
-3. (Optional) Globally enable metrics by setting `SIGNALFX_METRICS_ENABLED`:
-    ```bash
-    SIGNALFX_METRICS_ENABLED=true
-    ```
+3. Set `SIGNALFX_ENV` to specify the environment for the service in APM:
+   ```bash
+   SIGNALFX_ENV="yourEnvironment"
+   ```
 
-4. (Optional) Disable context propagation. The wrapper currently supports only
+4. If you didn't already specify span tags with the `tags` parameter, add add
```

Comment:
"add add"

Reply:
In general, a prefer to point to the other repo doc in case we change something in the future.